data/mods/moderngen1/scripts.ts

```diff
@@ -3,14 +3,15 @@ export const Scripts: ModdedBattleScriptsData = {
 	gen: 1,
 	init() {
 		for (const i in this.data.Pokedex) {
+			(this.data.Pokedex[i] as any).gen = 1;
 			(this.data.Pokedex[i] as any).gender = 'N';
 			(this.data.Pokedex[i] as any).eggGroups = null;
```

Comment:
This probably used to depend on `pokedex.ts` existing.
```suggestion
			this.modData('Pokedex', i).gen = 1;
			this.modData('Pokedex', i).gender = 'N';
			this.modData('Pokedex', i).eggGroups = null;
```
(there may well be a way to avoid the duplication)